### PR TITLE
Garima_Solihullsfc

### PR DESCRIPTION
**Pull request by Garima2412**
Please review and provide feedback

### CODE REVIEW

File: Garima_Solihullsfc/Resource/Pages/HomePage.robot [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Iterate through "${Letter}" Language from list and print the Titles
+      FOR               ${language}     IN              @{ListOfLanguage}
+
+        ${CurrentLanguage}=     get Element Attribute   ${language}      title
+        Click Element                                   ${language}
+        log to console                                  ${CurrentLanguage}
+#        ${languageDDUpdated}=       Replace string      ${languageDDUpdated}    <Language>      ${CurrentLanguage}
+#        wait until element is visible       ${languageDDUpdated}        timeout=30
+        ${PageTitle}=       Get Title
+        log to console                                  ${PageTitle}
```
Comment (narendrashakya) [inline Garima_Solihullsfc/Resource/Pages/HomePage.robot:44]: It only printing the "English" Title on each iteration. Need to impliment wait and print the title.
**Title getting print on each Title :** Solihull Sixth Form College: Post 16 education, A Levels and BTECs — Solihull Sixth Form College

image

**Review by narendrashakya — CHANGES_REQUESTED** [review]

File: Garima_Solihullsfc/solihullsfc.robot [hunk 2]
```diff
@@ -0,0 +1,14 @@
+*** Settings ***
```
Comment (narendrashakya) [inline Garima_Solihullsfc/solihullsfc.robot]: **Test** directory is not present, feature file need to be present in Test Directory.

**Review by narendrashakya — CHANGES_REQUESTED** [review]
"Test" folder is not present

File: Garima_Solihullsfc/Resource/Pages/HomePage.robot (hunk 1 above)
Comment (Garima2412) [reply Garima_Solihullsfc/Resource/Pages/HomePage.robot:44]: Updated

Comment (Garima2412) [reply Garima_Solihullsfc/Resource/Pages/HomePage.robot:44]: Updated - working now 
result

File: Configuration/Env.robot [hunk 3]
```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Library                     OperatingSystem
+Resource                    ../Pages/Adhoc.robot
```
Comment (narendrashakya) [inline Configuration/Env.robot]: Adhoc robot file is under **Resource** directory, resource path missing this directory. It giving below error.

Garima_Solihullsfc\Configuration\Env.robot' on line 3: Resource file '..\Pages\Adhoc.robot' does not exist.
**Correct path will be :** ../Resource/Pages/Adhoc.robot

**Review by narendrashakya — CHANGES_REQUESTED** [review]
Please update the correct path under ENV robot file.

**Review by narendrashakya — APPROVED** [review]
Approved 👍🏼

File: Garima_Solihullsfc/Resource/Pages/HomePage.robot (hunk 1 above)
Comment (mssquaretech) [reply Garima_Solihullsfc/Resource/Pages/HomePage.robot:44]: Wow.. keep it up

**Review by mssquaretech — CHANGES_REQUESTED** [review]
This branch contains multiple folders like Configuration, Resource, Test and Garima_Solihullsfc.
Expected : only one folder should be there - Garima_Solihullsfc.
Please remove other folders. 
Rest all is fine

**Review by mssquaretech — APPROVED** [review]

**Review by mssquaretech — COMMENTED** [review]
Changes done.. ready to merge

**Pull request merged**